<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewwindow</class>
 <widget class="QWidget" name="viewwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QPushButton" name="btn_refresh">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_refresh{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_refresh:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>行程号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>发车时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>剩余座位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>不退票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>优惠幅度</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_modify">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_modify{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_modify:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;查询输入&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="btn_find">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_find{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_find:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btn_delete">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_delete{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_delete:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="btn_quit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_quit{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_quit:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_add">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_add{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_add:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="btn_book">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_book{background:#66b1ff;color:#FFFFFF;border-radius:3px;}
QPushButton#btn_book:pressed{background:#3a8ee6;color:#FFFFFF}</string>
     </property>
     <property name="text">
      <string>订票</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
